### DISCUSSION TITLE
Respiration signal from PPG

### DISCUSSION THREAD

**Discussion by Borthakur123** [neuropsychology/NeuroKit#1037] [General]
I am wondering if we can use https://rpanderson-neurokit2.readthedocs.io/en/latest/examples/edr.html#ECG-Derived-Respiration-(EDR)-Analysis , ECG-Derived Respiration (EDR) Analysis technique for PPG too?

**Comment by DominiqueMakowski** [comment]
in theory yes, as iirc it's based on the heart rate, regardless of whether it comes from ecg or ppg
